avocado/utils/dmesg.py

```diff
@@ -106,7 +109,11 @@ def collect_errors_by_level(output_file=None, level_check=5):
                       verbose=False, shell=True)
     if out.exit_status == 0:
         err = "Found failures in dmesg"
-        dmsg_log = "dmesg log:\n%s" % out.stdout_text
+        if skip_error:
+            dmsg_log = skip_dmesg_msg(out.stdout_text, skip_error)
```

Comment:
About `skip_error`: Here you receive a `string`, and `skip_dmesg_msg()` is a `list`. Since this is a "list of dmesg messages for remove" we need to change this signature to a list too or convert this to a list as I suggested in my previous comment.

I would prefer the first approach (make this a list too).

Reply:
skip_error: is list i forget to change here.